ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-ref.yaml

```diff
@@ -3,7 +3,7 @@ ref:
   from_image:
     namespace: open-cluster-management
     name: builder
-    tag: go1.16-linux
+    tag: go1.17-linux
```

Comment:
As cnf-tests has already switched to go 1.18 why not not switch to that ?

Reply:
it doesn't exist yet for open-cluster-management/builder image we use, i tried it here https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/29558/rehearse-29558-periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g/1537408968882655232/build-log.txt

Reply:
let's merge this one to get a working CI and I will open another pr to update the image